apps/services/university-gateway/backend/src/app/modules/application/model/application.ts

```diff
@@ -0,0 +1,109 @@
+import { ApiProperty } from '@nestjs/swagger'
+import {
+  Column,
+  CreatedAt,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript'
+import { ProgramTable } from '../../program/model'
+import { University } from '../../university/model'
+import {
+  ApplicationStatus,
+  ModeOfDelivery,
+} from '@island.is/university-gateway-lib'
+
+@Table({
+  tableName: 'application',
+})
+export class Application extends Model {
```

Comment:
You can see example here:https://github.com/island-is/island.is/blob/main/apps/services/sessions/src/app/sessions/session.model.ts

This will help you when you start using the database model classes to create, query, update etc it will be typed from the model type.

```
export class ApplicationService {
  constructor(
    @InjectModel(Application)
    private readonly applicationModel: typeof Application,
  ) {}

  async createApplication(
    applicationDto: CreateApplicationDto,
    user: User,
  ): Promise<Application> {
    this.applicationModel.create({
    // The type is missing here and the create method doesn't know which properties can be used here.
    })
  }
```